roofit/roofitcore/inc/RooFit/TestStatistics/LikelihoodGradientWrapper.h

```diff
@@ -37,12 +38,22 @@ class LikelihoodGradientWrapper {
    LikelihoodGradientWrapper(std::shared_ptr<RooAbsL> likelihood,
                              std::shared_ptr<WrapperCalculationCleanFlags> calculation_is_clean, std::size_t N_dim,
                              RooMinimizer *minimizer);
+   LikelihoodGradientWrapper(std::shared_ptr<RooAbsL> likelihood,
+                             std::shared_ptr<WrapperCalculationCleanFlags> calculation_is_clean, std::size_t N_dim,
+                             RooMinimizer *minimizer,
+                             std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets,
+                             std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets_save);
    virtual ~LikelihoodGradientWrapper() = default;
    virtual LikelihoodGradientWrapper *clone() const = 0;
 
    static std::unique_ptr<LikelihoodGradientWrapper>
    create(LikelihoodGradientMode likelihoodGradientMode, std::shared_ptr<RooAbsL> likelihood,
           std::shared_ptr<WrapperCalculationCleanFlags> calculationIsClean, std::size_t nDim, RooMinimizer *minimizer);
+   static std::unique_ptr<LikelihoodGradientWrapper>
+   create(LikelihoodGradientMode likelihoodGradientMode, std::shared_ptr<RooAbsL> likelihood,
+          std::shared_ptr<WrapperCalculationCleanFlags> calculationIsClean, std::size_t nDim, RooMinimizer *minimizer,
+          std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets,
+          std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets_save);
```

Comment:
```suggestion
   static std::unique_ptr<LikelihoodGradientWrapper>
   create(LikelihoodGradientMode likelihoodGradientMode, std::shared_ptr<RooAbsL> likelihood,
          std::shared_ptr<WrapperCalculationCleanFlags> calculationIsClean, std::size_t nDim, RooMinimizer *minimizer,
          std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets=nullptr,
          std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets_save=nullptr);
```
Same suggestion as for the constructor.

Reply:
Done.